tez-common/src/main/java/org/apache/tez/common/TezUtilsInternal.java

```diff
@@ -74,14 +78,36 @@ public final class TezUtilsInternal {
 
   private TezUtilsInternal() {}
 
-  public static ConfigurationProto readUserSpecifiedTezConfiguration(String baseDir) throws
-      IOException {
+  public static ConfigurationProto readUserSpecifiedTezConfiguration(String baseDir) throws IOException {
     File confPBFile = new File(baseDir, TezConstants.TEZ_PB_BINARY_CONF_NAME);
     try (FileInputStream fis = new FileInputStream(confPBFile)) {
       return ConfigurationProto.parseFrom(fis);
     }
   }
 
+  public static Configuration readTezConfigurationXml(InputStream is) throws IOException {
+    Configuration configuration = new Configuration();
+    if (is != null) {
+      configuration.addResource(is);
+    }
+    return configuration;
+  }
+
+  public static ConfigurationProto loadConfProtoFromText() throws IOException {
```

Comment:
which method call this? this shows as unused. Removing it doesn't make any test fail either

Reply:
right, this is supposed to be added by [TEZ-4007](https://issues.apache.org/jira/browse/TEZ-4007) or [TEZ-4008](https://issues.apache.org/jira/browse/TEZ-4008), removing it now